ArduCopter/motors.pde

```diff
@@ -68,12 +68,12 @@ static void arm_motors_check()
     }
 }
 
-// auto_disarm_check - disarms the copter if it has been sitting on the ground in manual mode with throttle low for at least 15 seconds
+// auto_disarm_check - disarms the copter if it has been sitting on the ground in manual mode for at least 15 seconds
```

Comment:
Yep. I generally try to keep parameters like "15 seconds" out of my comments. The comment should be reworded to avoid specifying a time.